packages/fx-core/src/question/create.ts

```diff
@@ -181,10 +181,9 @@ function projectTypeQuestion(): SingleSelectQuestion {
     ProjectTypeOptions.bot(Platform.CLI),
     ProjectTypeOptions.tab(Platform.CLI),
     ProjectTypeOptions.me(Platform.CLI),
-    isOfficeXMLAddinEnabled()
-      ? ProjectTypeOptions.officeXMLAddin(Platform.CLI)
-      : ProjectTypeOptions.outlookAddin(Platform.CLI),
+    ProjectTypeOptions.officeXMLAddin(Platform.CLI),
```

Comment:
You need to rebuild fx-core package and commit the changes in `src/question/` folder.

Reply:
Got it. Will do. Thanks!